charts/renku/values.yaml

```diff
@@ -333,10 +333,10 @@ gitlab:
   password: gitlabadmin
 
   ## Gitlab image
-  # image:
+  image:
   #   pullPolicy: IfNotPresent
-  #   repository: gitlab/gitlab-ce
-  #   tag: latest
+    repository: gitlab/gitlab-ce
+    tag: 11.7.11-ce.0
```

Comment:
what about using 11.8.7-ce.0 ?

Reply:
I'd rather not since I just deployed it on dev ~1 hour ago :)